mem0/vector_stores/azure_ai_search.py

```diff
@@ -82,8 +116,8 @@ def create_col(self):
             SearchField(
                 name="vector",
                 type=vector_type,
-                searchable=True,
-                vector_search_dimensions=vector_dimensions,
+                searchable=True, # Tehnically don't need this on SearchField but leaving for visibility
+                vector_search_dimensions=self.embedding_model_dims,
                 vector_search_profile_name="my-vector-config",
             ),
             SimpleField(name="payload", type=SearchFieldDataType.String, searchable=True),
```

Comment:
This could be a "SearchableField", right? Given that this is a JSON document, not sure in what cases searching as if it were unstructured search will work unless we format searches too.

Reply:
IIRC - I ran into issues using "SearchableField" as I think that's designed for "text" fields only or something, I'm just aligning to the SDK defaults and how the other samples are written: 
 https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/search/azure-search-documents/samples/sample_vector_search.py